packages/medusa/src/loaders/express.ts

```diff
@@ -56,5 +58,10 @@ export default async ({ app, configModule }: Options): Promise<Express> => {
     res.status(200).send("OK")
   })
 
+  // Events caching middleware should be applied globally
+  // to ensure custom endpoints are covered
+  app.use(invalidateCachedEvents())
```

Comment:
todo: The naming looks like we are always invalidating the cache and the processing it. I would suggest to name it to something along the line of `invalidateCachedEventsOnError`